appUITests/Data/AccountsListTestData.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  AccountsListTestData.swift
+//  GoldenSampleUITests
+//
+//  Created by Backbase on 15/01/2024.
+//
+
+import Foundation
+
+struct AccountsListTestData{
+    //WARNING : Please adjust this based on the test data available
+    static let defaultAccountName: String = "Checking Account USD 7299"
+    static let defaultAccountNumberEnds: String = "7299"
+    static let defaultBalance: String = "11,309.73"
```

Comment:
I'd suggest to add some formatters to golden sample too, namely about amount and date, because their values depend on the region, which is set on the simulator and if that is not taken into account during assertions, tests will fail on some simulators, while pass on others. Examples:

Date:
- region UK: "9 Jan 2024"
- region US: "Jan 9, 2024"

Amount:
- region UA: "12 345,99"
- region US: "12,345.99"

Reply:
IMO this should be covered in a common test library instead in the golden sample project, to decided when we gonna use the test common libraries in golden example will  require further discussion. 
also this PR is not a one time implementation, more PR will come with changes and improvements.  @ashchupliak need your advise.

Reply:
I like the proposal, but in current scenario do we validate dates? If there is not usage then this could be postponed